test/web-platform-tests/wpt-server.js

```diff
@@ -33,13 +35,14 @@ exports.start = async ({ toUpstream = false } = {}) => {
 
   const configArg = path.relative(path.resolve(wptDir), configPath);
   const args = ["./wpt.py", "serve", "--config", configArg];
-  const subprocess = childProcess.spawn("python", args, {
+  subprocess = childProcess.spawn("python", args, {
     cwd: wptDir,
     stdio: ["inherit", "pipe", "pipe"]
   });
 
   subprocess.stdout.filter(nonSpammyWPTLog).pipe(process.stdout);
   subprocess.stderr.filter(nonSpammyWPTLog).pipe(process.stderr);
+  subprocess.stderr.on("data", terminateWptOnKeyError);
 
   return new Promise((resolve, reject) => {
     subprocess.on("error", e => {
```

Comment:
In what cases is this error expected to be triggered?
It seems that the format of `new Error()` is wrong in the first place.

Reply:
I think we should add an error listener outside the function.
I'll push 1 more commit to fix this.